playbooks/roles/vault/templates/config.hcl.j2

```diff
@@ -14,10 +14,12 @@ listener "tcp" {
     tls_require_and_verify_client_cert = "{{ "true" if vault_tcp_tls_require_and_verify_client_cert else "false" }}"
 }
 
+{% if vault_consul_acl_token is defined %}
```

Comment:
Tokens are optional, so although internally we do specify it, we should probably use something like `vault_consul_storage: <bool>` for deciding to render this or not instead.

Reply:
I'll do that: better be explicit about it. :+1: